io.fairyproject.modules/platform.core/core-legacy/src/main/java/io/fairyproject/redis/RedisService.java

```diff
@@ -63,9 +65,11 @@ public boolean shouldInitialize() {
     @SneakyThrows
     @PreInitialize
     public void preInit() {
+        Fairy.getLibraryHandler().downloadLibraries(true, Library.REDISSON);
```

Comment:
This shouldn't be needed if you have `library("org.redisson:redisson-all:3.13.6")` in build.gradle

Reply:
Yeah, I see. 😉 I removed it.